pebblo/app/config/config_validation.py

```diff
@@ -92,3 +92,16 @@ def validate_config(config_dict):
         for error in validation_errors:
             logger.error(error)
         sys.exit(1)
+
+
+def validate_cache_dir(cred_yaml):
+    dirs = cred_yaml.get("reports").keys()
+    if "cacheDir" in dirs and "outputDir" in dirs:
+        raise Exception("Either 'cacheDir' or 'outputDir' should be there in config")
```

Comment:
Technically this is correct, but it does not convey that which one is correct `cacheDir` or `outputDir`.

Reply:
Will update the message accordingly